pkg/deployer/helm/helm.go

```diff
@@ -87,6 +90,47 @@ func New(log logr.Logger, helmconfig *helmv1alpha1.Configuration, kubeClient cli
 	}, nil
 }
 
+func createOCIClient(ctx context.Context, log logr.Logger, client client.Client, item *lsv1alpha1.DeployItem, config *helmv1alpha1.Configuration, componentsRegistryMgr *componentsregistry.Manager) (ociclient.Client, error) {
```

Comment:
we have the structure that non-exposed helper functions are by below exposed functions in a file.
Can you put the helper functions to the bottom of the file.

Reply:
Done